src/components/Error/ErrorNotification.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { Error } from '../../types/Todo';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: Dispatch<SetStateAction<Error>>;
```

Comment:
You can keep it simpler
```suggestion
  setErrorMessage: (error: Error) => void
```

or even more specific:
```suggestion
  clearError: () => void
```